lesson02-variables-datatype/src/main/java/Exercise/Bai07.java

```diff
@@ -0,0 +1,45 @@
+package Exercise;
+
+import java.util.Random;
+
+public class Bai07 {
+
+	public static void main(String[] args) {
+		Random rd = new Random();
+
+		int a[] = new int[5];
+		int b[] = new int[5];
+		for (int i = 0; i < a.length; i++) {
+			a[i] = rd.nextInt(10) + 0;
```

Comment:
Probably you had a wrong understanding a bout req
MUST handle to a void the duplicate here, when you random them